crates/rolldown/tests/esbuild/loader/loader_file_relative_path_css/artifacts.snap

```diff
@@ -1,18 +1,25 @@
 ---
 source: crates/rolldown_testing/src/integration_test.rs
+snapshot_kind: text
 ---
 # Assets
 
+## assets/image-6tcw8vpN.png
+
 ## entries_entry.css
 
 ```css
 div {
-	background: url(../images/image.png);
+	background: url(assets/image-6tcw8vpN.png);
 }
 
 ```
 ## entries_entry.js
 
 ```js
 
+//#region images/image.png
+var image_default = "assets/image-6tcw8vpN.png";
+
+//#endregion
```

Comment:
Not sure here why need to generate the js file.

Reply:
https://github.com/rolldown/rolldown/pull/2598#discussion_r1828673482, as I said this, the redundant file is generated because the asset file related ecma view will codegen whatever, could be polished  in the future, But I think it is not related to this pr, since this pr only handle `url()` dependency